js/table.js

```diff
@@ -0,0 +1,47 @@
+// Filling table cells
+
+function updateTable(newCurrencyObject, updatingObject, changingRateObject) {
```

Comment:
This function could be split into smaller ones: `displayNewCurrencyInTable`, `updateCurrencyInTable`, `changeRateInTable`